### PR TITLE
EPMRPP-56784: Report launches/ suites with logs (with standard level log) Log view

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: lib/reporter.js [hunk 1]
```diff
@@ -51,14 +51,41 @@ class Reporter {
...
+        if (err) {
+            throw err;
+        }
+
+        const type = args.messages && args.messages[0];
+
+        switch (type) {
+            case 'launch':
+                if (this.launchLogs !== null) {
+                    args.messages.slice(1).forEach(message => this.launchLogs.push({ level: args.level, message }));
```
Comment (ElenaRomanchuk) [inline lib/reporter.js:76]: Is it possible to collect the time of each log message and send it later to the API? Could we restore the sequence of log messages? It seems a user will expect to have logs in proper order in the results.

Comment (KatsiarynaTatarynovich) [reply lib/reporter.js:76]: Thank you for the comment. I added the additional parameter `time`.

**Review by ElenaRomanchuk — APPROVED** [review]

File: lib/reporter.js [hunk 2]
```diff
@@ -202,18 +241,36 @@ class Reporter {
             .promise.catch(errorHandler);
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    getTime () {
+        return new Date().valueOf();
```
Comment (AmsterGet) [inline lib/reporter.js:246]: Why not to use `this.client.helpers.now`?

Comment (KatsiarynaTatarynovich) [reply lib/reporter.js:246]: The client's method is not quite suitable, since `getTime` method works faster than the client's one

**Pull request merged**